relay-profiling/src/sample.rs

```diff
@@ -233,6 +261,10 @@ pub fn expand_sample_profile(payload: &[u8]) -> Result<Vec<Vec<u8>>, ProfileErro
             }
         });
 
+        if new_profile.profile.samples.is_empty() {
+            return Err(ProfileError::NotEnoughSamples);
+        }
```

Comment:
This is going to drop the entire profile when one of the transactions in it has no samples right?